examples/chart/teleport-cluster/README.md

```diff
@@ -4,6 +4,9 @@ This chart sets up a single node Teleport cluster.
 It uses a persistent volume claim for storage.
 Great for getting started with Teleport.
 
+It can also deploy HA Teleport clusters on EKS, GKE and other Kubernetes clusters.
```

Comment:
Nit: we should be consistent in the use of the Oxford comma. (Not necessarily partial to this but we use this pretty much everywhere else.)

Reply:
I hate Oxford commas, but if we're using them elsewhere I'll do it!

Reply:
I appreciate your agreement, correcting this trivial issue, and adding the comma! 😀